### PR TITLE
NIAD-24: added docker run script and updated local running readme

### PR DESCRIPTION
**Pull request by michael-kainos**

### CODE REVIEW

**Review by pageholder — DISMISSED** [review]

File: docker-build-components-example.sh [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+: '
+Make a copy of this file and rename to docker-build-components.sh
+as the renamed file is in .gitignore. This will avoid commiting the file
+with certificate information.
+'
+echo "Exporting environment variables"
+#Put your party key here (OpenTest):
+export MHS_SECRET_PARTY_KEY=""
```
Comment (mattd-kainos) [inline docker-build-components-example.sh]: do all of these variables need to be exported or none of them?

File: docker-build-components-example.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+#Put your Endpoint issuing subCA certificate and your Root CA certificate here (OpenTest)
+MHS_SECRET_CA_CERTS=""
+
+#Stops containers, builds images and starts containers
+echo "Stopping running containers"
+docker-compose -f docker-compose.yml -f docker-compose.lb.override.yml stop;
+
+echo "Build and starting containers"
+docker-compose -f docker-compose.yml -f docker-compose.lb.override.yml up -d --build
```
Comment (mattd-kainos) [inline docker-build-components-example.sh]: Personal preference really, but I tend to run "attached" or without the -d flag. Sometimes the logs in the console are useful.

File: docker-build-components-example.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+: '
+Make a copy of this file and rename to docker-build-components.sh
```
Comment (bartek-sarul) [inline docker-build-components-example.sh]: I don't like the idea of making this example shell file
Env vars can be set in bash/zsh startup script
This file would then have only docker commands and don't have to be in gitignore

File: docker-build-components-example.sh (hunk 1 above)
Comment (michael-kainos) [reply docker-build-components-example.sh]: Should all be exported as they are used in the docker compose

File: docker-build-components-example.sh (hunk 2 above)
Comment (michael-kainos) [reply docker-build-components-example.sh]: Makes sense, i only do this so a user does not try to carry on using the shell and accidently exits from their start up. Thoughts on this as a use case?

File: mhs/running-mhs-adaptor-locally.md [hunk 4]
```diff
@@ -51,4 +51,31 @@ used to facilitate health checks in a load balanced environment.
...
+
+## Running MHS components and load balancer with docker
+
+To run the MHS services using docker containers follow the steps below:
+
+1. Make a file in the root call `export-env-vars-and-run-mhs-docker.sh` this name must match to be excluded from git
+2. Populate the file with the following information:
+  ```sh
+#!/bin/bash
+echo -e "\033[1;32mExporting environment variables\033[0m"
```
Comment (bartek-sarul) [inline mhs/running-mhs-adaptor-locally.md]: Hard to understand at first look
Please add variables as in https://github.com/nhsconnect/integration-adaptors/pull/193/files#diff-b42d9fb95f574b12b6c1a033fa91dddbR3

**Review by bartek-sarul — CHANGES_REQUESTED** [review]

**Review by bartek-sarul — DISMISSED** [review]

**Review by bartek-sarul — APPROVED** [review]

**Pull request merged**